runtime/onert/backend/train/ops/LossMeanSquaredErrorLayer.cc

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "LossMeanSquaredErrorLayer.h"
+#include "OperationUtils.h"
+
+#include <cker/train/operation/Loss.h>
+
+namespace onert
+{
+namespace backend
+{
+namespace train
+{
+namespace ops
+{
+
+void LossMeanSquaredErrorLayer::configure(const IPortableTensor *y_pred,
+                                          const IPortableTensor *y_true, IPortableTensor *output,
+                                          IPortableTensor *back_prop_y_pred)
+{
+  LossLayer::configure(y_pred, y_true, output, back_prop_y_pred);
+}
+
+void LossMeanSquaredErrorLayer::forward(bool)
+{
+  // TODO Implement this
```

Comment:
@jyoungyun What does "Implement this" mean? FLOT32 case seems implemented. Maybe, is this comment for else case? If so, throw seems enough.

Reply:
Ｙｅｓ，　ｔｈｉｓ　ｆｕｎｃｔｉｏｎ　ｎｅｅｄｓ　ｔｏ　ｂｅ　ｉｍｐｌｅｍｅｎｔｅｄ　ｆｏｒ　ｄｉｆｆｅｒｅｎｔ　ｄａｔａ　ｔｙｐｅｓ．

Reply:
Yes, this function needs to be implemented for different data types.

Reply:
I considered this comment was there before you implement `FLAT32`, and left by mistake. Because as I wrote in the above, 

>    throw std::runtime_error("LossMeanSquaredErrorLayer: unsupported data type");

already speaks the same thing.